tests/konnector_structure.coffee

```diff
@@ -10,7 +10,9 @@ describe 'Check all konnectors', ->
     listOfKonnectors = fs.readdirSync konnectorsDirectory
     listOfLocales = fs.readdirSync localesDirectory
 
-    listOfKonnectors.forEach (filename) ->
+    # Remove hidden files (a.k.a. thoses starting with '.')
+    listOfKonnectors.filter (filename) -> filename.indexOf('.') != 0
```

Comment:
Actually I wanted to do that (JS habit) but Coffee yelled at me. It looks like it's automatically converted to `!==`